### PR TITLE
NOISSUE-Updated kubernetes.md

### PR DESCRIPTION
**Pull request by Hasan98-git**
Updated nginx_internal.mtls.tls to nginxInternal.mtls.tls and nginx_internal.intermediate_crt to nginxInternal.intermediateCrt in kubernetes.md.

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

**Comment by manuio** [comment]
@Hasan98-git DCO is failling. Please, follow the contributing guidelines (https://github.com/mainflux/docs/blob/master/CONTRIBUTING.md) and sign your commits.

**Review by drasko — DISMISSED** [review]
LGTM

File: docs/kubernetes.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ The following table lists the configurable parameters and their default values.
 | defaults.replicaCount                | Replicas of MQTT adapter, Things, Envoy and Authn                          | 3            |
 | defaults.natsPort                    | NATS port                                                                  | 4222         |
 | defaults.jaegerPort                  | Jaeger port                                                                | 6831         |
-| nginx_internal.mtls.tls              | TLS secret which contains the server cert/key                              |              |
-| nginx_internal.mtls.intermediate_crt | Generic secret which contains the intermediate cert used to verify clients |              |
+| nginxInternal.mtls.tls              | TLS secret which contains the server cert/key                              |              |
+| nginxInternal.mtls.intermediateCrt | Generic secret which contains the intermediate cert used to verify clients |              |
```
Comment (dborovcanin) [inline docs/kubernetes.md]: Please check alignment here and sign off your commits.

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**